src/robusta/integrations/kubernetes/base_triggers.py

```diff
@@ -189,12 +189,32 @@ def should_fire(self, event: TriggerEvent, playbook_id: str, build_context: Dict
 
         return True
 
+    @classmethod
+    def __patch_k8s_object(cls, obj: Dict[Any, Any], model_class: type) -> None:
```

Comment:
It feels off seeing this here when we have a `patch.py` file where we do all the hikaru/kubernetes client patches

Reply:
yes, it doesn't work with patching
this is for handling a change event, where you parse the object from yaml
the error is from the `from_dict` method, which we can't patch